lib/oli/delivery.ex

```diff
@@ -118,7 +128,8 @@ defmodule Oli.Delivery do
           nrps_context_memberships_url: NRPS.get_context_memberships_url(lti_params),
           customizations: customizations
         })
-      {:ok, %Section{}} = Sections.create_section_resources(section, publication)
+
+      {:ok, %Section{} = section} = Sections.create_section_resources(section, publication)
```

Comment:
Most of the changes in this file are auto-formatting, but this line is the key change that fixes the root_section_resource nil issue